Adapt "Choose units for plotting " HowTo for ISU units (#6593)
gammapy/gammapy: docs/user-guide/howto.rst

        ax.yaxis.set_units(u.Unit("erg cm-2 s-1"))
        fp.plot(ax=ax, sed_type="e2dnde")

    By default Gammapy uses bracket notation for axis labels (e.g. ``Energy [TeV] ``).

>     According to International System of Units the formally correct notation is ``quantity / unit``

Reviewer: ```suggestion
    According to International System of Units the recommended notation is ``quantity / unit``
```

    (e.g. ``Energy / TeV`` ).
    To apply this , override the axis labels manually after plotting :

>     .. code::

Reviewer: An empty line is necessary here, see eg lines 105-107 of this file

        ax.yaxis.set_units(u.Unit("erg cm-2 s-1"))
        fp.plot(ax=ax, sed_type="e2dnde")

    By default, Gammapy uses bracket notation for its axis labels (e.g. ``Energy [TeV]``).
    According to the International System of Units (`ISU <https://www.bipm.org/documents/20126/41483022/SI-Brochure-9-EN.pdf>`_) the recommended notation is ``quantity / unit``
    (e.g. ``Energy / TeV``).
    To adopt these standards one can override the axis labels manually after creating the plot:

    .. code::

        ax.set_xlabel(r"$E$ / eV")
        ax.set_ylabel(r"$E^2 \, \mathrm{d}N/\mathrm{d}E$ / (erg cm$^{-2}$ s$^{-1}$)")

    Alternatively, you can change this behaviour globally for all plots by updating
    the default label template. An example of this for an estimator plot is:

    .. code::

        from gammapy.estimators import points
        points.core.DEFAULT_LABEL_TEMPLATE = "{quantity} / {unit}"

>     Similarly adapt `gammapy.modeling.models.spectral.DEFAULT_LABEL_TEMPLATE` for spectral models.

Reviewer 2: ```suggestion
    Similarly, one can adapt `gammapy.modeling.models.spectral.DEFAULT_LABEL_TEMPLATE` for spectral models.
```

        ax.yaxis.set_units(u.Unit("erg cm-2 s-1"))
        fp.plot(ax=ax, sed_type="e2dnde")

    By default Gammapy uses bracket notation for axis labels (e.g. ``Energy [TeV]``).
    According to International System of Units the recommended notation is ``quantity / unit``
    (e.g. ``Energy / TeV`` ).

>     To apply this,override the axis labels manually after plotting:

Reviewer 3: Can you add the link towards the pdf somewhere in this text?

>     To apply this,override the axis labels manually after plotting:

Reviewer 2: ```suggestion
    By default, Gammapy uses bracket notation for its axis labels (e.g. ``Energy [TeV]``).
    According to the International System of Units (ISU) the recommended notation is ``quantity / unit``
    (e.g. ``Energy / TeV`` ).
    To adopt these standards one can override the axis labels manually after creating the plot:
```

    .. code::

        ax.set_xlabel(r"$E$ / eV")
        ax.set_ylabel(r"$E^2 \, \mathrm{d}N/\mathrm{d}E$ / (erg cm$^{-2}$ s$^{-1}$)")

    Alternatively you can change this behaviour globally for all plots by updating
    the default label template.

>     For modeling plots:

Reviewer 2: ```suggestion
    Alternatively, you can change this behaviour globally for all plots by updating
    the default label template. An example of this for a modelling plot is:
```

    .. code::

        from gammapy.modeling.models import spectral
        spectral.DEFAULT_LABEL_TEMPLATE = "{quantity} / {unit}"

    For estimator plots:

    .. code::

        from gammapy.estimators import points
        points.core.DEFAULT_LABEL_TEMPLATE = "{quantity} / {unit}"

> 

Reviewer 2: ```suggestion
```

I would remove this, as we just want a small example to show it is possible for all plots. 
In general, we should try to keep the `HowTo` as small examples and not make them too large

Reviewer: Then maybe show the estimator example and say, "similarly adapt `gammapy.modeling.models.spectral.DEFAULT_LABEL_TEMPLATE` for spectral models"

This needs to be clarified, because depending on the order in which one calls the plot functions, the axis label is printed accordingly, and the users have no way of knowing where all this `DEFAULT_LABEL_TEMPLATE` is defined and used internally
